cloudhub-2/modules/ROOT/pages/ch2-patch-updates.adoc

```diff
@@ -27,6 +27,29 @@ When running Mule in a xref:ch2-clustering.adoc[cluster], a rolling update with
 
 To prevent this kind of error, use the Recreate update strategy.
 
+
+[[self-upgrade-schedule]]
+== Schedule Infrastructure Upgrades (Optional)
+
+Schedule infrastructure upgrades earlier than the scheduled patching process, between the Release Available and the Auto-Update dates stated in the xref:release-notes::cloudhub-2/cloudhub-2-runtimes-release-notes.adoc[monthly updates table]. This way, you can choose a date in the future that better suits your organization's needs.
+
+On average, this upgrade takes between thirty to sixty minutes to complete. If your upgrade is queued, it can take longer than thirty minutes. The impact of the upgrade process on running applications is consistent with the auto-patching behavior described in the previous sections.
+
+You can modify or delete your scheduled upgrade if it is not already in progress. If you cancel the schedule and do not create another or opt in to an earlier upgrade, the upgrade occurs during the mandatory patching window.
```

Comment:
It would be good to mention that if the status is 'Queued', then it's an indication that the upgrade has begun and is in progress.

Reply:
```suggestion
You can modify or delete your scheduled upgrade if it is not already in progress, as indicated by the `Queued` status. If you cancel the schedule and do not create another or opt in to an earlier upgrade, the upgrade occurs during the mandatory patching window.
```